### PR TITLE
API ML logon 

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <43951184+IgorCATech@users.noreply.github.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)

Adding a sub-section on the 'Log in to API ML" feature.

Tech.implementation PR https://github.com/zowe/vscode-extension-for-zowe/pull/1062 

:heart:Thank you!

### CODE REVIEW

**Comment by IgorCATech** [comment]
I'm still not sure what the use case for Log out from API ML

**Comment by balhar-jakub** [comment]
The log out in the SSO case invalidates the JWT token. 
If the JWT was issued by zOSMF then it asks zOSMF to invalidate 
On top of that API ML keep cache of invalidated JWT token, which when queried against the API ML returns invalid.

**Review by zdmullen — COMMENTED** [review]
So this is alright, but as Richelle noted in stand-up, we'll have to respond to the comments left by Billie in the PR and make more general mention of the process being available for profiles that use tokens. We'll hold off on this for a bit. Thanks

File: docs/user-guide/ze-profiles.md [hunk 1]
```diff
@@ -111,4 +111,20 @@ Connect your service profile with a base profile and token.
...
+### Log in to API Mediation Layer
+
+If the token for your base profile is no longer valid, you can log in again to get a new token with the Log in to API Mediation Layer feature.
+
+**Note:** The feature is only available for base profiles.
+
+**Follow these steps:**
+
+1. Open Zowe Explorer.
+2. Right-clikc your profile.
```
Comment (crawr) [inline docs/user-guide/ze-profiles.md:125]: small typo here Right-click

File: docs/user-guide/ze-profiles.md [hunk 2]
```diff
@@ -111,4 +111,20 @@ Connect your service profile with a base profile and token.
...
+3. Select the **Log in to API Mediation Layer** option.
```
Comment (crawr) [inline docs/user-guide/ze-profiles.md:126]: The new option name is **Log in to Authentication Service**.

Although, the only option for now is via the API Mediation Layer, other extenders may have a different authentication service. Perhaps this should be mentioned somewhere?

Comment (zdmullen) [reply docs/user-guide/ze-profiles.md]: I think that this comment should go in the release notes for the version, as well as in the doc somewhere. We'll just have to track it and update when other services are available.

**Review by zdmullen — APPROVED** [review]
Looks good. Let's just make sure we add the note about the auth service currently available in the release notes. Thanks!

File: docs/user-guide/ze-profiles.md [hunk 3]
```diff
@@ -111,4 +111,33 @@ Connect your service profile with a base profile and token.
...
+
+1. Open Zowe Explorer.
+2. Right-click your profile.
+3. Select the **Log in to Authentication Service** option.
+
+   You will be prompted to enter your username and password beforehand.
+
+The token is stored in the default base profile .yaml file.
+
+If you do not want to store your token, request from the server to end the session of your token. Use the **Log in to Authentication Service** feature to invalidate the token.
```
Comment (crawr) [inline docs/user-guide/ze-profiles.md:135]: Small typo here, it should be Log out

**Comment by IgorCATech** [comment]
Can this be merged now? @crawr

**Comment by IgorCATech** [comment]
Also, release notes for the Zowe general RN page were added

**Review by crawr — APPROVED** [review]

**Pull request merged**